internal/node/node_launcher.sh

```diff
@@ -125,11 +126,17 @@ NODE_OPTIONS=()
 ALL_ARGS=(TEMPLATED_args $NODE_REPOSITORY_ARGS "$@")
 for ARG in "${ALL_ARGS[@]}"; do
   case "$ARG" in
+    --bazel_node_modules_manifest=*) MODULES_MANIFEST="${ARG#--bazel_node_modules_manifest=}" ;;
     --node_options=*) NODE_OPTIONS+=( "${ARG#--node_options=}" ) ;;
     *) ARGS+=( "$ARG" )
   esac
 done
 
+# Link the first-party modules into node_modules directory before running the actual program
+if [[ -n "$MODULES_MANIFEST" ]]; then
+  "${node}" "${link_modules_script}" "${MODULES_MANIFEST}"
```

Comment:
Is there going to be some way to indicate that this dosen't need ot be run?

Reply:
I don't see how we could know that without looking around the disk a little, which is the job of the linker. I think we can make the linker fast enough? It's a small program and can do efficient file I/O

Reply:
Sounds good